beacon-chain/p2p/discovery.go

```diff
@@ -58,6 +58,10 @@ func (s *Service) RefreshENR() {
 		log.WithError(err).Error("Could not initialize persistent subnets")
 		return
 	}
+	if err := initializePersistentColumnSubnets(s.dv5Listener.LocalNode().ID()); err != nil {
```

Comment:
Why doing this into the `RefreshENR` function?
(It seems it does not have a link with the ENR, since there is no corresponding `columnSubnet` entry in the ENR.)

Reply:
It is easier to do it here as it is called periodically. Also this is called in line with our att subnets,. we possibly might rotate assignments so having the current epoch is useful. 

In the future once we are not required to put it in the enr anymore we can change the naming of the method